app/gui2/src/util/ast/string.ts

```diff
@@ -0,0 +1,17 @@
+const mapping: Record<string, string> = {
+  '\b': '\\b',
+  '\f': '\\f',
+  '\n': '\\n',
+  '\r': '\\r',
+  '\t': '\\t',
+  '\v': '\\v',
+  '"': '\\"',
+  "'": "\\'",
+  '`': '``',
+}
+
+/** Escape a string so it can be safely spliced into an interpolated (`''`) string.
+ * NOT USABLE to insert into raw strings. Does not include quotes. */
```

Comment:
I would mention here what exactly you mean by string - is it Enso string or JS string?